client/src/components/PricingTotal.jsx

```diff
@@ -77,52 +88,51 @@ class PricingTotal extends React.Component {
   }
 
   getOccupanyTaxPercentage(city) {
-    //TO-DO;
-    let percent = 0.075;
+    let percent = 0.075; //hard-coded
```

Comment:
as the value will not change, perhaps use `const`, plus convention of upper case name with underscores for constant values, and a more descriptive name, e.g. `SALES_TAX`